web/src/components/njwds-layout/SidebarPageLayout.tsx

```diff
@@ -23,45 +24,52 @@ export const SidebarPageLayout = ({
   outlineBox = true,
   stackNav,
   belowBoxComponent,
+  titleOverColumns,
 }: SidebarPageLayoutProps): ReactElement => {
   const isLargeScreen = useMediaQuery(MediaQueries.desktopAndUp);
   const { Config } = useConfig();
 
   const backButton = (
-    <Link href={ROUTES.dashboard} passHref>
-      <a
-        href={ROUTES.dashboard}
-        data-testid="back-to-dashboard"
-        className="usa-link fdr fac margin-top-3 margin-bottom-3 usa-link-hover-override desktop:margin-top-0 desktop:margin-bottom-4"
-        onClick={analytics.event.task_back_to_roadmap.click.view_roadmap}
-      >
-        <div className="circle-3 bg-accent-cool-darkest icon-blue-bg-color-hover">
-          <Icon className="text-white usa-icon--size-3">arrow_back</Icon>
-        </div>
-        <div className="margin-left-2 margin-y-auto text-accent-cool-darkest font-sans-xs usa-link text-blue-color-hover">
-          {Config.taskDefaults.backToRoadmapText}
-        </div>
-      </a>
-    </Link>
+    <nav>
```

Comment:
[dust] Are there additional aria-related attributes to designate this as a secondary nav that would be appropriate here?

Reply:
not really? We could add an Aria-label for added clarity but its not strictly necessary, otherwise everything else that I'm aware of would be implied by these tags alone. 

Also walked through with a screen reader and it seemed good. 

Did you have a suggestion?

Reply:
I will check with amanda for a aria-label of secondary nav